bool reached
bool rotated
bool is_stuck
float32 driving_for
float32 idle_for